haas/cli.py

```diff
@@ -124,6 +117,8 @@ def serve_networks():
     """Start the HaaS networking server"""
     from haas import model, deferred
     from time import sleep
+    server.register_drivers()
```

Comment:
Might be reasonable --- we could also pass through the create parameter to the init_db function, which would let us use it everywhere (and maybe rename it to just server_init, since it would no longer only work in the api server).

@gsilvis, what do you think?